### PR TITLE
Remove MySQL and add MariaDB installation instructions

### PR DESCRIPTION
**Pull request by sandoche2k**

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -64,6 +64,11 @@ Install dependencies available from binary packages:
 ```sh
 sudo yum install perl-Module-Install perl-IO-CaptureOutput perl-String-ShellQuote
 ```
+*  Only for CentOS 7 *
+
+```sh
+sudo yum install perl-Plack-Test
+```
```
Comment (matsduf) [inline docs/Installation.md:71]: Plack::Test is not a direct dependency. If it is an indirect dependency we should not have to list it here. Furthermore, is it really true that it is only needed for CentOS 7?

Comment (mattias-p) [reply docs/Installation.md:71]: Plack::Test is part of the Plack CPAN distribution which we do depend on. For some reason the CentOS packaging of the Plack CPAN distribution is split into two packages: perl-Plack and perl-Plack-Test. So I'm not surprised to see perl-Plack-Test among our dependencies. But Plack seems to be split the same way on both CentOS 7 and 8, and it's not obvious to me why perl-Plack-Test is only needed on one of them.

Comment (sandoche2k) [reply docs/Installation.md:71]: Yes, needed only for CentOS7

**Review by mattias-p — APPROVED** [review]

**Review by matsduf — APPROVED** [review]

**Comment by matsduf** [comment]
@sandoche2k, please merge.

**Pull request merged**